src/averageAges.js

```diff
@@ -15,11 +15,22 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const diedInСertainСentury = (person) => (
+    !century || century === Math.ceil(person.died / 100)
+  );
+  const men = people.filter(
+    person => person.sex === 'm' && diedInСertainСentury(person)
+  );
+  const menAgesSum = men.reduce(
+    (sum, person) => sum + person.died - person.born,
+    0
+  );
+
+  return menAgesSum / men.length;
```

Comment:
you do the same on calculateWomenAverageAge. You can create separate function and you it here and on calculateWomenAverageAge